lib/modules/us_core_guidance/us_core_r4_capability_statement_sequence.rb

```diff
@@ -77,7 +77,13 @@ def json_formats
           'http://hl7.org/fhir/us/core/StructureDefinition/pediatric-weight-for-height',
           'http://hl7.org/fhir/us/core/StructureDefinition/us-core-pulse-oximetry',
           'http://hl7.org/fhir/us/core/StructureDefinition/us-core-smokingstatus',
-          'http://hl7.org/fhir/StructureDefinition/vitalsigns'
```

Comment:
What does adding these profiles do?

Reply:
Warnings will be generated for any of these profiles which are not listed as supported in the CapabilityStatement.

Reply:
Makes sense.